.eslintrc.js

```diff
@@ -9,4 +9,9 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
 	'prettier',
   ],
+  rules: {
+    // disabling this since it is used everywhere in this repo
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Do we have a ticket to fix this?

Reply:
I thought about trying to fix it, but I'm not sure it is possible. There are too many places in the code that require the use of `any`. I would rather disable it and trust the team to not use it where it isn't required.